docs/guides/access-groups.mdx

```diff
@@ -4,27 +4,60 @@ description: How to create and manage access groups for your organization's acco
+## Create an IBM Quantum Administrators access group
+After setting up an account for your organization, it's recommended that you create an  IBM Quantum Administrators access group. This access group lets other users  create and manage instances, and manage user access for the Qiskit Runtime service.
 
-When you create this access group, include the following policies.  You can copy and paste the code into the JSON entry feature on the IAM console. Alternatively, you can make all of the selections on the user interface, as described in the [next section.](#ui)
+When you create this access group, include the following policies:
 
-- All IAM Account Management services (Administrator), required for inviting users, managing access groups and creating access policies
+- Qiskit Runtime service - Grant access to manage all IBM Quantum instances in the account and view account usage analytics.
+  - **Manager** service access role
+  - **Administrator** platform management access role
+- All account management services - Grant access to list all the resource groups in the account.
+  - **Viewer** platform management access role
+- All IAM Account Management services - Grant access to invite users, manage access groups, and create access policies.
+  - **Administrator** platform management access role
+
+
+<Admonition type="note">
+  Users with the `viewer` platform management role on "all account management services" can also view services such as billing. To prevent this extra view access, use the IBM Cloud CLI to give them access to just Resource groups:
+  ```cli
+  ibmcloud iam access-group-policy-create <group name> --roles Viewer --resource-type resource-group
+  ```
+</Admonition>
+
+Follow these examples to create an Administrators access group by using the IBM Cloud CLI or console.
+
+<span id="cli"></span>
+### Use the IBM Cloud CLI
+
+To create an _access group_ by using the CLI, use the [`ibmcloud iam access-group-create`](https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_commands_iam#ibmcloud_iam_access_group_policy_create) command.
+
+```cli
+ibmcloud iam access-group-create GROUP_NAME [-d, --description DESCRIPTION]
+```
+
+To create an access group _policy_ by using the CLI, use the [`ibmcloud iam access-group-policy-create`](https://cloud.ibm.com/docs/cli?topic=cli-ibmcloud_commands_iam#ibmcloud_iam_access_group_policy_create) command.
+
+```cli
+ibmcloud iam access-group-policy-create GROUP_NAME {-f, --file @JSON_FILE | --roles ROLE_NAME1,ROLE_NAME2... [--service-name SERVICE_NAME] [--service-instance SERVICE_INSTANCE] [--region REGION] [--resource-type RESOURCE_TYPE] [--resource RESOURCE] [--resource-group-name RESOURCE_GROUP_NAME] [--resource-group-id RESOURCE_GROUP_ID]}
+```
+
+You can use the following JSON code to create policies for an Administrators access group:
 
 ```json
```

Comment:
this JSON should be placed below "All IAM Account Management services (administrator)"

Reply:
this is the JSON you referenced missing here: https://github.com/Qiskit/documentation/pull/2912#issuecomment-2773717432

looks like in this rewrite the text was moved to the bottom on the bullet points but the JSON was left in the same place